joi-validations/chat.validations.js

```diff
@@ -60,6 +60,7 @@ const ChatValidation = {
       anon_user_info_color_name: string.required(),
       anon_user_info_emoji_code: string.required(),
       anon_user_info_emoji_name: string.required(),
+      context: Joi.optional(),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `context` field has been made optional, which seems fine. However, it would be better to specify the type of `context` for clarity and validation purposes. If `context` is a string, you can use `Joi.string().optional()`. If it's an object, you can use `Joi.object().optional()`.
```diff
- context: Joi.optional(),
+ context: Joi.string().optional(), // or Joi.object().optional() based on the expected type
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->